src/Campaigns/Blocks/CampaignDonors/render.php

```diff
@@ -36,8 +35,8 @@
 if ($sortBy === 'top-donors') {
     $query->select(
         'donorIdMeta.meta_value as id',
-        'SUM(amountMeta.meta_value) as amount',
-        'donorName.meta_value as name'
+        'SUM(CAST(amountMeta.meta_value AS DECIMAL)) AS amount',
+        'MAX(donorName.meta_value) AS name'
```

Comment:
Can you explain why you used `max` here? I'm not sure I'm following this.

Reply:
I saw the comment in Jira, hmm...
Try to change the group by to `donorName.meta_value` instead of `donorIdMeta.meta_value` and see what happens.